server.rb

```diff
@@ -37,26 +43,67 @@ def language(id)
       sender_id = messaging["sender"]["id"]
       text = messaging["message"]["text"]
 
-      unless @@users[sender_id]
-        Bot.new.send_message(sender_id, "To what language would you like me to translate? I know a lot")
-        Bot.new.send_message(sender_id, "Please respond with '/language'. For example, '/English'")
-      end
-
-      @@users[sender_id] ||= "English"
-
-      if text[0] == "/"
-        @@users[sender_id] = text[1..-1]
-      end
+      # unless @@users[sender_id]
```

Comment:
delete old code. git remembers. future programmers don't need to. great programmers know writing code is easy so deleting it (or rewriting it to be better) is also easy.